portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,27 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Azfaar's Portfolio</title>
+	<link rel="stylesheet" href="style.css">
+	<script src="script.js"></script>
+</head>
+
+<body>
+	<div id="content">
+		<img src="/images/azfaar.jfif" alt="Azfaar's picture" width="200" height="200">
+		<h2>Hi, my name is</h2>
+		<h1>Azfaar Qureshi</h1>
+		<h3>Welcome to my portfolio.</h3></br>
+		<p>I'm currently a 3rd year Computer Engineering student at the University of Waterloo.
+			I have experience with Frontend, Backend and Production engineering.
+			I'm graduating in April 2022 and am currently searching for my next internship.
+			Take a look at my resume and previous experiences below!</p>
+		<p>Click here to get a random greeting:</p>
+		<button onclick="addRandomGreeting()">Hello</button>
+		<div id="greeting-container"></div>
+	</div>
+</body>
+
+</html>
```

Comment:
Nit: Please add a blank line at the end of the file. It doesn't really matter, but GitHub adds a red marker on the end of any file without one.

Reply:
Huh, for some reason vscode linting removes the newline. I'll get rid of the linter then so I can add the new line

Reply:
done!